tests/e2e/helpers/base.js

```diff
@@ -94,8 +118,8 @@ export function runBaseTests (br) {
     const initialUrl = await getUrl();
 
     // Set Cookie by page navigation, wait for cookie
-    const goForward = await isRightToLeft() ? nav.desktop.goPrevious : nav.desktop.goNext;
-    await t.click(goForward)
+    const goNext = getNext(nav, await bookPageProgression());
```

Comment:
For consistency, you could instantiate a `pageProgression` variable like you do in line 143 and pass it to `getNext`

Reply:
done